avm/res/sql/server/elastic-pool/main.bicep

```diff
@@ -0,0 +1,92 @@
+metadata name = 'SQL Server Elastic Pool'
+metadata description = 'This module deploys an Azure SQL Server Elastic Pool.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Required. The name of the Elastic Pool.')
+param name string
+
+@description('Conditional. The name of the parent SQL Server. Required if the template is used in a standalone deployment.')
+param serverName string
+
+@description('Optional. Tags of the resource.')
+param tags object?
+
+@description('Optional. Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Capacity of the particular SKU.')
+param skuCapacity int = 2
+
+@description('Optional. The name of the SKU, typically, a letter + Number code, e.g. P3.')
+param skuName string = 'GP_Gen5'
+
+@description('Optional. The tier or edition of the particular SKU, e.g. Basic, Premium.')
+param skuTier string = 'GeneralPurpose'
+
+@description('Optional. The number of secondary replicas associated with the elastic pool that are used to provide high availability. Applicable only to Hyperscale elastic pools.')
+param highAvailabilityReplicaCount int = -1
+
+@description('Optional. The license type to apply for this elastic pool.')
+@allowed([
+  'BasePrice'
+  'LicenseIncluded'
+])
+param licenseType string = 'LicenseIncluded'
+
+@description('Optional. Maintenance configuration resource ID assigned to the elastic pool. This configuration defines the period when the maintenance updates will will occur.')
+param maintenanceConfigurationId string = ''
+
+@description('Optional. The storage limit for the database elastic pool in bytes.')
+param maxSizeBytes int = 34359738368
+
+@description('Optional. Minimal capacity that serverless pool will not shrink below, if not paused.')
+param minCapacity int = -1
```

Comment:
Same comment as for the database module

Reply:
done: https://github.com/Azure/bicep-registry-modules/pull/637/commits/8ca7dc449f84bcee7942c205cf4daf1ebb88a752